jaxrs/WFLY-11737_http_proxy.adoc

```diff
@@ -0,0 +1,55 @@
+= WFLY-11737 / EAP7-1200 RESTEasy support for setting HTTP proxy by using JAX-RS API
```

Comment:
> Can you please adjust ... the title to reference the community issue only.

According to @darranl previous message, title shouldn't contains reference to EAP7 jira ...